.github/CODEOWNERS

```diff
@@ -88,6 +95,10 @@ integration/testdata/TestCnbIntegration/        @SAP/jenkins-library-cnb
 /pkg/abap/                                      @SAP/jenkins-library-abap
 /pkg/abaputils/                                 @SAP/jenkins-library-abap
 
+/pkg/blackduck/                                 @SAP/oss-team
```

Comment:
could you move this closer to the other section, so that all of them are together?
Rest looks good